site/docs/0.0/examples/album/index.html

```diff
@@ -1,7 +1,7 @@
 ---
 layout: examples
 title: Album example
-extra_css: "album.css"
+<!--extra_css: "album.css"-->
```

Comment:
I don't think it's valid to use HTML comments like these within the [YAML front-matter](https://jekyllrb.com/docs/front-matter/).  I'm getting errors like this when I try running `npm run docs-compile` locally:

```
YAML Exception reading /arizona-bootstrap/site/docs/0.0/examples/blog/index.html: (<unknown>): could not find expected ':' while scanning a simple key at line 4 column 1
YAML Exception reading /arizona-bootstrap/site/docs/0.0/examples/album/index.html: (<unknown>): did not find expected key while parsing a block mapping at line 2 column 1
```

Maybe we should just delete these lines instead?

Reply:
Or alternatively comment these lines with `#`.

Reply:
Commented with `#`